src/js/view/view.js

```diff
@@ -404,7 +404,11 @@ function View(_api, _model) {
             },
             doubleClick: () => _controls && api.setFullscreen(),
             move: () => _controls && _controls.userActive(),
-            over: () => _controls && _controls.userActive()
+            over: (evt) => {
+                if (_controls && (evt.sourceEvent.fromElement.classList.length === 0 || evt.sourceEvent.fromElement.classList[0].indexOf('jw-nextup') === -1)) {
```

Comment:
Had to also add check for if the classList array was empty, otherwise there was an exception being thrown if the mouse event was sourcing from an element without any classes due to `classList[0]`.

Reply:
Use `evt.sourceEvent.fromElement.className.indexOf('jw-nextup') === -1`. That's just a string of all classes, and it's an empty string when there are no classes - unless we're dealing with a different nodeType that doesn't where classes are undefined. In that case `(evt.sourceEvent.fromElement.className || '').indexOf('jw-nextup') === -1`